### ISSUE TITLE
Only Mana and Artifice village structures generating

### ISSUE THREAD

**Issue by LezChap** [FTBTeam/FTB-Presents-Direwolf20-1.16#154]
* **Modpack Version**: 1.4.1

* **Issue**: Mana and Artifice appears to be preventing other mod's village structures (and thus villagers) from generating in villages.  I believe this issue started in 1.3, but I may be wrong on that.  In earlier versions of the pack, Pneumaticraft (and other probably others, but I can't recall which ones) modded village structures would generate with villages.  I believe starting in 1.3, the only village structures that appear to be generating in villages are the Mana and Artifice ones (which, in some cases, you can find 4-5+ per village).

* **Is it Repeatable?**: Yes
Confirmed by generating flat worlds of all all the major village biomes (Plains, Savanna, Tiaga, and Desert, to make sure it wasn't a biome thing) and flew around until I found a handful of villages in each.

* **Mod/s Affected**: Mana and Artifice, Pneumaticraft, probably others

* **Known Fix**: none known

**Comment by uecasm** [comment]
I was surprised when I found a desert village I hadn't been to before that had at least 6 Artifice villagers and rune forge houses, and nothing else other than vanilla.  I guess this explains it.

**Comment by Sunekaer** [comment]
Im not able to recreate this issue, what seed, any changes to the pack, do you also have this issue in a newer version of the pack?

**Comment by uecasm** [comment]
I'm using DW20 1.5.1 with the LP seed and I saw it in my world.  However I'm not sure exactly which version of the modpack I was using when those particular chunks were generated.

Creating a new world today (random seed) and locating a village showed it had several different structure types (including one mana & artifice house, but only one).  Visiting a second village (a desert village, like the one where I had found five previously) had two, but also had other structures.

However, generating a new world with DW20's seed and going to the village at 302 65 251 (which is where I found the large number of mana & artifice houses in my own world) also reveals a strangely high number of mana & artifice villagers -- although this time they're all manaweavers, whereas in my world they were runeforgers.  (There's 3 manaweaving buildings plus at least one more M&A "ritualist" building, although there are a few regular structures as well.)

So... I don't entirely know what that means, whether it's better or still an issue.

**Comment by LezChap** [comment]
I was using random seeds before, in version 1.4.1, with no mod or config changes.  I'd see Vanilla structures, and Mana and Artifice structures, but nothing else.

I've updated to 1.5.1, and just did another test in a random seed, superflat world (more villages)  Found 2 villages with Immersive Engineering structures of the 4 or 5 I /located and teleported to.  Don't know if that means it's "fixed" or if I has just really bad RNG (I tested around 5-6 villages in like 4 or 5 random seed worlds of various biomes) before.
